test/jdk/javax/sound/midi/spi/MidiDeviceProvider/ExpectedNPEOnNull.java

```diff
@@ -32,6 +32,7 @@
 /**
  * @test
  * @bug 8143909
+ * @key sound
  * @author Sergey Bylokhov
```

Comment:
What exception causes this(and others) tests to fail if the hardware device is not configured? please add it to the PR of jbs.

Reply:
It can cause these tests to hang in native methods if device is not configured or not accessible in latest macOS. We need finer control as to which tests needs not to be executed on such machines.